cli/compose/loader/loader.go

```diff
@@ -725,6 +727,14 @@ func transformSize(value interface{}) (interface{}, error) {
 	panic(errors.Errorf("invalid type for size %T", value))
 }
 
+func transformIsolation(value interface{}) (interface{}, error) {
+	str, ok := value.(string)
+	if !ok {
+		return value, errors.Errorf("invalid type %T for isolation", value)
+	}
+	return container.Isolation(strings.ToLower(str)), nil
```

Comment:
Missed this in my previous review; can you remove the `strings.ToLower()` here as well?